.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,17 @@
+name: Deploy docs
+
+on:
+  # Runs on pushes targeting the `main` branch. Change this to `master` if you're
+  # using the `master` branch as the default branch.
+  push:
+    branches: [main]
+    paths:
+      - "docs/**"
+      - ".github/workflows/**" # define the concrete paths on which a change triggers this action, e.g. backend/**
+
+  # Allows you to run this workflow manually from the Actions tab
```

Comment:
Why do we need `workflow_dispatch` when docs get automatically updated on `main` push. Where is the effective difference?